tests/environment_var/main.tf

```diff
@@ -0,0 +1,18 @@
+module "stdout" {
```

Comment:
also the directory should be `tests/environment/main.tf`

Reply:
updated